src/sql/strong-oracle/create.sql

```diff
@@ -0,0 +1,332 @@
+BEGIN
```

Comment:
What we probably want is to move these to a `drop.sql` and instead use what you suggested here.